packages/renderer/src/lib/kube/KubernetesDashboard.svelte

```diff
@@ -0,0 +1,143 @@
+      serviceCount = services.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextIngresses.subscribe(ingresses => {
+      ingressCount = ingresses.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextRoutes.subscribe(routes => {
+      routeCount = routes.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextPersistentVolumeClaims.subscribe(pvcs => {
+      pvcCount = pvcs.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextConfigMaps.subscribe(configMaps => {
+      configMapCount = configMaps.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextSecrets.subscribe(secrets => {
+      secretCount = secrets.length;
+    }),
+  );
+});
+
+onDestroy(async () => {
+  unsubscribers.forEach(unsubscriber => unsubscriber());
+});
+</script>
+
+<NavPage searchEnabled={false}  title="About Kubernetes">
+  <svelte:fragment slot="additional-actions">
+    {#if !welcome}
+    <KubeApplyYamlButton />
+    {/if}
+  </svelte:fragment>
+
+  <svelte:fragment slot="bottom-additional-actions">
+    <div class="flex grow justify-end">
+      <KubernetesCurrentContextConnectionBadge />
+    </div>
+  </svelte:fragment>
+
+  <div class="flex min-w-full h-full" slot="content">
+    {#if welcome}
+        <KubernetesEmptyPage />
+    {:else}
+    <div class="flex flex-col space-y-4 min-w-full overflow-y-auto">
+        <div class="flex flex-col gap-4 p-5">
+            <span>Here you can manage and interact with Kubernetes clusters with features like connect to clusters,
+            view workloads like deployments and services.</span>
+
+        <span>Get up and running by clicking one of the menu items!</span>
+
+        <Link class="place-self-start" on:click={() => window.openExternal('https://podman-desktop.io/docs/kubernetes')}>Kubernetes documentation</Link>
+    </div>
+
+<div class="flex flex-col gap-4 bg-[var(--pd-content-card-bg)] grow p-5">
+    {#if currentContextName}
+    <div class="text-xl">Metrics</div>
+    <div class="grid grid-cols-4 gap-4">
+        <KubernetesDashboardResourceCard type='Nodes' Icon={NodeIcon} count={nodeCount} link='/kubernetes/nodes'/>
+        <KubernetesDashboardResourceCard type='Deployments' Icon={DeploymentIcon} count={deploymentCount} link='/kubernetes/deployments'/>
+        <KubernetesDashboardResourceCard type='Services' Icon={ServiceIcon} count={serviceCount} link='/kubernetes/services'/>
+        <KubernetesDashboardResourceCard type='Ingresses & Routes' Icon={IngressRouteIcon} count={ingressCount + routeCount} link='/kubernetes/ingressesRoutes'/>
+        <KubernetesDashboardResourceCard type='Persistent Volume Claims' Icon={PvcIcon} count={pvcCount} link='/kubernetes/persistentvolumeclaims'/>
+        <KubernetesDashboardResourceCard type='Config Maps & Secrets' Icon={ConfigMapSecretIcon} count={configMapCount+secretCount} link='/kubernetes/configmapsSecrets'/>
```

Comment:
```suggestion
        <KubernetesDashboardResourceCard type='ConfigMaps & Secrets' Icon={ConfigMapSecretIcon} count={configMapCount+secretCount} link='/kubernetes/configmapsSecrets'/>
```

ConfigMaps is one word

Reply:
Fixed.

Reply:
@cdrage PR has passed all tests, approved, and set to auto-merge but isn't. Seems to still be hung on this even though doesn't show at top right? This was fixed, please check again and review or merge. Thanks.